tests/xtimer_msg_receive_timeout/main.c

```diff
@@ -30,27 +26,27 @@
 #include "xtimer.h"
 #include "timex.h"
 
-#define TEST_PERIOD (100000LU)
+#define TEST_PERIOD     (1*US_PER_SEC) /* 1 sec in US */
+#define TEST_COUNT      (10)
```

Comment:
```10LU``` please. C uses the left hand type to determine result type, and a number literals default to ```int```. So if later down you're using "10 + offset", the result will overflow at 2^16 on 16bit platforms. *edit* Actually it should overflow at ```2^15```.